test/common/stats/recent_lookups_speed_test.cc

```diff
@@ -52,20 +53,20 @@ class RecentLookupsSpeedTest {
   Envoy::Stats::RecentLookups recent_lookups_;
 };
 
-static void BM_LookupsMixed(benchmark::State& state) {
+static void bmLookupsMixed(benchmark::State& state) {
```

Comment:
elsewhere we have been using this pattern:
```
// NOLINTNEXTLINE(readability-identifier-naming)
static void BM_AddResponsesRealSymtab(benchmark::State& state) {
```
I really don't care that much personally, and don't know what significant the `BM_` prefix has to tooling or anything, but should we change everything to the `bmLookupMixed` pattern? Maybe we should add a note to the style guide about what our policy should be.

Reply:
The style suggested by the google benchmarking libraries is the BM_ stuff.  I somewhat prefer following the style rather than add NOLINT lines everywhere.  We really ought to cleanup clang errors and move away from diff mode.  clang-tidy errors due to pre-existing issues that are surfaced every time we touch a file for whatever reason are a burden.